client/utils/previewEntry.js

```diff
@@ -175,6 +176,6 @@ if (_report) {
     paths.forEach((path) => {
       newMessage = newMessage.replaceAll(path, window.objectPaths[path]);
     });
-    _report.apply(window.p5, [newMessage, method, color]);
+    p5Call.apply(window.p5, [newMessage, method, color]);
```

Comment:
Previously the `apply` was on `window.p5._report`.  Now it is on `window.p5`.  So that's a mistake.  I would keep a `_report` variable but define it inside the `if (p5Call)` branch.

Reply:
Thanks for looking further into this!! I actually didn't realize there were so many issues about this already 😂 

I think for now, only changing `window.p5` to `window.p5?.report` makes sense to me. 

I'll change this PR as progress towards the issue, since it seems like we need to change where `previewScripts` is loading, but I'm not sure where the best placement might be for it yet.